packages/babel-helper-simple-access/src/index.js

```diff
@@ -0,0 +1,87 @@
+import * as t from "babel-types";
+
+export default function simplifyAccess(path: NodePath, bindingNames) {
+  path.traverse(simpleAssignmentVisitor, {
+    scope: path.scope,
+    bindingNames,
+    seen: new WeakSet(),
+  });
+}
+
+const simpleAssignmentVisitor = {
+  UpdateExpression: {
+    exit(path) {
+      const { scope, bindingNames } = this;
+
+      const arg = path.get("argument");
+      if (!arg.isIdentifier()) return;
```

Comment:
This would be super (😉, you'll get it in a second) helpful for `super.prop++` and `this.#foo++`, too.

Reply:
At the moment this only accepts a list of identifier names, so that'd be tough, but we could certainly expand that functionality later.